lib/compiler.js

```diff
@@ -661,6 +661,7 @@ function resolveRelativePaths(relativePaths, appRootDir) {
 function getExcludedExtensions() {
   return {
     '.json': '.json',
+    '.map': '.map',
```

Comment:
Please add a comment explaining that this is a temporary workaround - see https://github.com/strongloop/loopback-boot/pull/245#issuecomment-329098622

> I think the `.map` entry should be also accompanied by a comment explaining that it's a partial fix for a bigger problem and pointing to https://github.com/strongloop/loopback-boot/pull/245#issuecomment-311052798

Reply:
Ah, yep, missed that. Cheers.